pptx/table.py

```diff
@@ -383,6 +384,109 @@ def vertical_anchor(self):
     def vertical_anchor(self, mso_anchor_idx):
         self._tc.anchor = mso_anchor_idx
 
+    @property
+    def borL(self):
+        """
+        Get Left Border instance containing all the formatting options.
+        """
+        return self._tc.tcPr.borL
+
+    @lazyproperty
+    def border_left(self):
+        return BorderFormat(self, 'left')
+
+    def get_or_add_borL(self):
+        return self._tc.tcPr.get_or_add_borL()
+
+    @property
+    def borR(self):
+        """
+        Get Right Border instance containing all the formatting options.
+        """
+        return self._tc.tcPr.borR
+
+    @lazyproperty
+    def border_right(self):
+        return BorderFormat(self, 'right')
+
+    def get_or_add_borR(self):
+        return self._tc.tcPr.get_or_add_borR()
```

Comment:
Are all these function (`get_or_add_borR`, `get_or_add_borL`, `get_or_add_borT`, etc..) necessary in _Cell?

Reply:
actually, they are not.  The are created dynamically which I just learned today.